src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingleadingasterisk/InputMissingLeadingAsteriskCorrect.java

```diff
@@ -0,0 +1,9 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingleadingasterisk;
+
+public class InputMissingLeadingAsteriskCorrect {
```

Comment:
No need for separate test case and input. Other test input also has correct examples.
Please remove it with test

Reply:
Done